sklearn/tests/test_kernel_approximation.py

```diff
@@ -31,12 +32,10 @@ def test_polynomial_count_sketch_raises_if_degree_lower_than_one(degree):
         ps_transform.fit(X, Y)
 
 
-@pytest.mark.parametrize("X", [X, csr_matrix(X)])
-@pytest.mark.parametrize("Y", [Y, csr_matrix(Y)])
 @pytest.mark.parametrize("gamma", [0.1, 1, 2.5])
-@pytest.mark.parametrize("degree", [1, 2, 3])
-@pytest.mark.parametrize("coef0", [0, 1, 2.5])
-def test_polynomial_count_sketch(X, Y, gamma, degree, coef0):
+@pytest.mark.parametrize("degree, n_components", [(1, 500), (2, 500), (3, 5000)])
```

Comment:
reducing the number of components makes it faster (but the assertions fails for degree=3)